app/controllers/nonsipp/memberdetails/SchemeMemberDetailsAnswersController.scala

```diff
@@ -67,11 +68,10 @@ class SchemeMemberDetailsAnswersController @Inject()(
   def onSubmit(srn: Srn, index: Max300, checkOrChange: CheckOrChange): Action[AnyContent] =
     identifyAndRequireData(srn).async { implicit request =>
       for {
-        updatedUserAnswers <- Future.fromTry(
-          request.userAnswers
-            .set(MemberStatus(srn, index), MemberState.Active)
-            .set(MemberDetailsCompletedPage(srn, index), SectionCompleted)
-        )
+        updatedUserAnswers <- request.userAnswers
+          .set(NewMember(srn, index))
```

Comment:
I'm just thinking how the ETMP memberDetails.memberDetailsType` value "Changed" would work here. Would it be just to not set  NewMember when the checkOrChange is change? (to be replaced in future with Mode.change mode)

Reply:
Yeah thats a good shout, I'll change it to only set on NormalMode